python/ql/src/experimental/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -325,29 +535,40 @@ abstract class DataFlowCall extends TDataFlowCall {
   /** Get the callable to which this call goes. */
   abstract DataFlowCallable getCallable();
 
-  /** Get the specified argument to this call. */
+  /**
+   * Gets the argument to this call that will be sent
+   * to the `n`th parameter of the callable.
+   */
   abstract Node getArg(int n);
 
   /** Get the control flow node representing this call. */
   abstract ControlFlowNode getNode();
 
   /** Gets the enclosing callable of this call. */
   abstract DataFlowCallable getEnclosingCallable();
+
+  /** Gets the location of this dataflow call. */
+  Location getLocation() { result = this.getNode().getLocation() }
 }
 
-/** Represents a call to a callable (currently only callable values). */
-class CallNodeCall extends DataFlowCall, TCallNode {
+/**
+ * Represents a call to a callable (currently only callable values).
+ * This excludes calls to bound methods, classes, and special methods.
```

Comment:
What are "special methods"? (I'm assuming things like `__str__` and `__add__`, but not entirely sure).

I see that yes, this is a standard python term, so all good I think :+1: 

https://docs.python.org/3/glossary.html#term-special-method